designer/client/src/components/graph/PanZoomPlugin.ts

```diff
@@ -189,18 +189,22 @@ export class PanZoomPlugin {
 
     private initPanMove = (paper: dia.Paper) => {
         const hammer = new Hammer(paper.el);
-        hammer.get("pan").set({ threshold: 2 });
-
-        paper.on("blank:pointerdown", (e) => {
-            this.disabledPan = e.target?.parentElement?.id !== "nk-graph-main";
+        hammer.get("pan").set({
+            threshold: 2,
+            direction: Hammer.DIRECTION_ALL,
```

Comment:
It fixes the issue with a pan timeout https://touk-jira.atlassian.net/browse/NU-1203